partner_communication/migrations/10.0.1.0.2/post-migration.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2018 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Nicolas Bornand
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate(use_env=True)
```

Comment:
Since you are never using the environment, you can remove this decorator, remove the openupgrade import, and the method signature will take cr as first parameter instead of env. Then you can simply use it in the code to do cr.execute(...)